Data/Helpers/Seed.cs

```diff
@@ -486,6 +486,34 @@ public static List<CallToActionOption> SeedCallToActionOptions()
                    };
         }
 
+        /// <summary>
+        ///     This method seeds call to actions.
+        /// </summary>
+        /// <returns>Returns a list of call to actions options that can be seeded into the database.</returns>
+        public static List<CallToAction> SeedCallToActions(List<Project> projects)
+        {
+            List<CallToAction> callToActions = new List<CallToAction>();
+
+            for(int i = 0; i < projects.Count; i++)
+            {
+                for(int j = 0; j < new Random().Next(0, 3); j++)
+                {
+                    Faker<CallToAction> fakeCallToAction = new Faker<CallToAction>()
+                                                           .RuleFor(c => c.OptionValue, f => f.Lorem.Word())
+                                                           .RuleFor(c => c.Value, f => f.Lorem.Word())
+                                                           .RuleFor(c => c.ProjectId, projects[i].Id);
+
+                    CallToAction callToAction = fakeCallToAction.Generate();
+
+                    callToActions.Add(callToAction);
+                }
+                
+            }
+
+            return callToActions;
+
+        }
+
```

Comment:
What's the purpose for seeding call to actions? I think seeding the call to action options is good enough? The users are not able to see (and use) these seeded call to actions.

Reply:
This was a mistake on my part to keep this in the code, this was more for testing locally on my end. This can be removed now.

Reply:
@RubenFricke I don't think it's bad to seed it, right? We are also seeding projects which are not used in production.